test/unit/utils.test.ts

```diff
@@ -0,0 +1,19 @@
+import { privateKeyToEncodedPublicKey } from "../../src/utils"
+import data from "../data/utilsData"
+describe("Unit Test Cases for utils", () => {
+    describe("privateKeyToEncodedPublicKey Testcases", () => {
+        test("Should pass for valid input for privateKey and compres is true", () => {
+            const encodedPublicKey = privateKeyToEncodedPublicKey(data.DUMMY_PRIVATE_KEY)
+            expect(typeof encodedPublicKey).toBe("string")
+        })
+        test("Should pass for valid input for privateKey and compress is false", () => {
+            const encodedPublicKey = privateKeyToEncodedPublicKey(data.DUMMY_PRIVATE_KEY, false)
+            expect(typeof encodedPublicKey).toBe("string")
```

Comment:
How this expect is different from the expect at line 7

Reply:
I thought of testing for the type instead of value because it may change if we change the test data

Reply:
but it would not give an assurance that the value we got is correct. There is a possibility of getting incorrect data but still this test case would pass because that data would still in `string` type. Recommend to validate data and its type both. 

Let's use the actual key that would act as the test key for the test.

Reply:
got it